### PR TITLE
Update Scala 2.13 to 2.13.15

### PR DESCRIPTION
**Pull request by piotrp**
## Describe your changes

We build Nu with Scala 2.13.12 and use Tapir built with Scala 2.13.13, which is incompatible - see SIP-51 for details. If we were using sbt 1.10.0+ our build would fail.

## Checklist before merge
- [ ] Related issue ID is placed at the beginning of PR title in \[brackets\] (can be GH issue or Nu Jira issue)
- [x] Code is cleaned from temporary changes and commented out lines
- [ ] Parts of the code that are not easy to understand are documented in the code
- [ ] Changes are covered by automated tests
- [ ] Showcase in dev-application.conf added to demonstrate the feature
- [ ] Documentation added or updated
- [ ] Added entry in _Changelog.md_ describing the change from the perspective of a public distribution user
- [ ] Added _MigrationGuide.md_ entry in the appropriate subcategory if introducing a breaking change
- [ ] Verify that PR will be squashed during merge

### CODE REVIEW

File: docs/Changelog.md [hunk 1]
```diff
@@ -81,6 +81,7 @@
   * [#7058](https://github.com/TouK/nussknacker/pull/7058) Add missing Flink TypeInformation for better serialization
   * [#7097](https://github.com/TouK/nussknacker/pull/7097) Flink base types registration mechanism
 * [#7021](https://github.com/TouK/nussknacker/pull/7021) Definitions service can return definition without UI config
+* [#7113](https://github.com/TouK/nussknacker/pull/7113) Updated Scala 2.13 to 2.13.15
```
Comment (coutoPL) [inline docs/Changelog.md:84]: what do you think about moving it to the "updated dependency" bag?

**Comment by piotrp** [comment]
Oops, forgot about flink-scala-2.13. Publishing is in progress, I will update this PR when Maven Central will start to see the new version.

**Comment by piotrp** [comment]
Done, now with updated flink-scala-2.13

**Review by coutoPL — APPROVED** [review]
LGTM

**Pull request merged**